community/cython/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=cython
+pkgver=0.29.6
+pkgrel=0
+pkgdesc="Cython is an optimising static compiler for both the Python & the extended Cython programming languages."
+url="https://cython.org"
+arch="all"
+license="Apache-2.0"
+_py2_deps="python2-dev"
+_py3_deps="python3-dev"
+options="!check" # several tests fail
+makedepends="py2-setuptools py3-setuptools $_py2_deps $_py3_deps"
+subpackages="${pkgname}2:_py2 ${pkgname}3:_py3 $pkgname-doc"
```

Comment:
The use of variables should be limited to places where there is a chance of a value to change. In this case i don't think pkgname will ever change (we already concluded pkgname should stay same). It will add complexity to read it, even if it's just minimal.

Reply:
I have replaced the `pkgname` variable with its explicit value in whole file.